src/lib/entropy/win32_stats/es_win32.cpp

```diff
@@ -72,7 +72,6 @@ void Win32_EntropySource::poll(Entropy_Accumulator& accum)
 
    if(!accum.polling_finished())
       {
-      size_t heap_lists_found = 0;
```

Comment:
This move changes the semantics of the poll I think. When here, the variable represented the total calls  to Heap32ListFirst, after move limits total calls to Heap32First.